RELEASE/scripts/sl_ascend/sl_combat.ash

```diff
@@ -2635,7 +2635,7 @@ string sl_edCombatHandler(int round, string opp, string text)
 
 	if(!contains_text(edCombatState, "curseofstench") && sl_have_skill($skill[Curse Of Stench]) && (my_mp() >= 35) && (get_property("stenchCursedMonster") != opp) && (get_property("sl_edCombatStage").to_int() < 3))
 	{
-		if($monsters[Bob Racecar, Cabinet of Dr. Limpieza, Dairy Goat, Dirty Old Lihc, Government Scientist,  Green Ops Soldier, Possessed Wine Rack, Pygmy Bowler, Pygmy Witch Surgeon, Quiet Healer, Racecar Bob, Writing Desk] contains enemy)
+		if($monsters[Bob Racecar, Cabinet of Dr. Limpieza, Dairy Goat, Dirty Old Lihc, Government Scientist,  Green Ops Soldier, Possessed Wine Rack, Pygmy Bowler, Pygmy Witch Surgeon, Quiet Healer, Racecar Bob, Writing Desk, Blue Oyster cultist] contains enemy)
```

Comment:
Just use `sl_wantToSniff` here instead of a hardcoded list. I'm updating `sniff.dat` with Ed specific sniffs now, so don't worry about that.

Reply:
... And some really important sniffs I apparently missed when converting to data driven monster properties. That or they just weren't being done before. (Specifically Blue Oyster cultist, Possessed Wine Rack, and GROPS, although I'm not sure if sl_ascend actually utilizes the smoke bombs, but eh, still nice to get)

Reply:
Done and thanks for pointing out that function (and creating it, the despaghettification is very much appreciated).